setup.py

```diff
@@ -52,6 +52,5 @@
         "Operating System :: Microsoft",
         'Topic :: Games/Entertainment :: Simulation',
         'Topic :: Scientific/Engineering :: Artificial Intelligence',
-        'Framework :: Robot Framework :: Tool'
```

Comment:
Could be replaced by 'Framework :: Robot Framework'

Reply:
> Hi,
> 
> The Robot Framework tag can be found on PyPi for packages such as **pybullet** and the **qi** software stack (for SoftBank Robotic's robots). This project uses **pybullet**, and emulates a part of the **qi** APIs, I would argue that this tag makes sense. Eventually, it could be replaced by 'Framework :: Robot Framework'

These projects use the trove classifiers not correctly as well. The classifiers are meant to be used for the test automation framework Robot Framework, not PyPi projects which address robots in general.